dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -78,15 +83,29 @@ public void bindTo(StatementExecution cs, int ind) throws SQLException{
         this.cs=cs;
         this.index=ind;    
         if (direction != INPUT){
-            cs.registerOutParameter(ind, getSqlType());
+            cs.registerOutParameter(ind, getSqlType(), direction == RETURN_VALUE);
         }
     }
 
+    private Object transformValue(Object value) throws SQLException {
```

Comment:
> I was now thinking of using TransformToJdbcAdapterCompatibleValue() as that's really why we're doing it.
> 
> Sound reasonable?

Yes. I think it could also be: `transformToJdbcCompatibleValue` or `toJdbcCompatibleValue`.